spec/elastic_apm/trace_context_spec.rb

```diff
@@ -111,6 +111,68 @@ module ElasticAPM
       end
     end
 
+    describe '#apply_headers' do
+      subject do
+        described_class.new.tap do |tp|
+          tp.trace_id = '1' * 32
+          tp.id = '2' * 16
+          tp.flags = '00000011'
+        end
+      end
+
+      context 'when prefixed disable', :intercept do
+        it 'yields twice' do
```

Comment:
What does it mean "yields twice"? `calls` is validated to be 1 below...?

Reply:
It was a lazy description. Updated it.